### PR TITLE
Define contexts for CocoaLumberjack

### PR DESCRIPTION
**Pull request by 0xced**
As recommended by [CocoaLumberjack logging contexts documentation](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/8adde11d0b16843cb45b81dc9b60d1430eb9b538/Documentation/CustomContext.md#example-2)

> This will allow application developers to easily manage the log statements coming from your framework.

### CODE REVIEW

File: Code/Support/RKLumberjackLogger.m [hunk 1]
```diff
@@ -75,7 +75,7 @@ + (void)logWithComponent:(_RKlcl_component_t)component
     [DDLog log:async
          level:componentLevel
           flag:flag
-       context:0 /* Could define a special value here to identify RestKit logs to any backend loggers */
+       context:0x524B5F00 + component
```
Comment (segiddins) [inline Code/Support/RKLumberjackLogger.m]: Should this be exposed as a constant anywhere?

Comment (0xced) [inline Code/Support/RKLumberjackLogger.m]: I don’t think a constant is needed, but some documentation would be welcome. I searched for logging documentation but found nothing. Where would be an appropriate place to document this context value? README.md? (That’s what I did for [XCDYouTubeKit](https://github.com/0xced/XCDYouTubeKit/tree/d3732863e73d697afcf769f0ea55b44c9068e9d3#logging))

I also opened [issue #529](https://github.com/CocoaLumberjack/CocoaLumberjack/issues/529) on the CocoaLumberjack project.

Comment (segiddins) [inline Code/Support/RKLumberjackLogger.m]: I'd just document it around RKLog somewhere?

**Comment by carllindberg** [comment]
The point of having a custom context would be so that custom logger code could actually use it -- why shouldn't it be a public constant?

**Pull request merged**